inst/rmarkdown/templates/trb/resources/template.tex

```diff
@@ -0,0 +1,80 @@
+% Repository:  https://github.com/chiehrosswang/TRB_LaTeX_tex
+%
+% Transportation Research Board conference paper template
+% version 4.0 Lite (updates made to be compatible in Overleaf and ShareLaTeX)
+%
+%
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbarticle}
+\usepackage{graphicx}
+\usepackage{booktabs}
+
+\newread\somefile
+\usepackage{xparse}
+
+\usepackage{natbib}
+\bibliographystyle{unsrtnat}
+\setcitestyle{round}
+
+% \usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
+% For TRB version hide links
+\usepackage[hidelinks]{hyperref}
```

Comment:
Would love to have advice. With a lot of this, there is a balance between "let the author override a default" and "just hard code what everyone will do."

Reply:
Done in b80d85d

Reply:
I added support for `hyperrefoptions` Pandoc's variable but keeping hidelinks the default

We can remove it and put 
````
hyperrefoptions: hidelinks
```

in the Rmd file to do it otherwise.